geoportal/c2cgeoportal_geoportal/__init__.py

```diff
@@ -607,9 +607,9 @@ def handle(event: InvalidateCacheEvent):  # pylint: disable=unused-variable
 
     c2cwsgiutils.index.additional_title = "<h2>GeoMapFish</h2>"
     c2cwsgiutils.index.additional_auth.extend([
-        '<p><a href="../tiles_admin/?{secret_qs}">TileCloud chain admin</a></p>',
-        '<p><a href="../invalidate?{secret_qs}">Invalidate the cache</a></p>',
-        '<p><a href="../memory?{secret_qs}">Memory status</a></p>',
```

Comment:
Shouldn't also me removed in the next c2cwsgiutils release?

Reply:
It's part of the list: https://github.com/camptocamp/c2cwsgiutils/issues/177